mindsdb/api/mysql/mysql_proxy/mysql_proxy.py

```diff
@@ -130,8 +133,57 @@ def isAuthOk(self, user, orig_user, password, orig_password):
             log.error(f'Check auth, user={user}: ERROR')
             log.error(traceback.format_exc())
 
+    def make_temp_cert(self):
```

Comment:
Can't we use library e.g. the one linked in the issue instead of doing all of this manually? Or is the fine-grain control required to get it looking exactly like the kind of certs MySQL works with?

Reply:
That easiest pure-python cert-generation what i can find